lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8441,6 +8449,26 @@ ClangImporter::Implementation::loadAllMembers(Decl *D, uint64_t extra) {
     loadAllMembersOfObjcContainer(D, objcContainer);
     return;
   }
+
+  auto namespaceDecl =
+      dyn_cast_or_null<clang::NamespaceDecl>(D->getClangDecl());
+  if (namespaceDecl) {
+    auto *enumDecl = cast<EnumDecl>(D);
+    for (auto ns : namespaceDecl->redecls()) {
```

Comment:
Yeah, we should definitely be filtering based on top-level Clang module here. Can you leave a FIXME?

Reply:
(Also, ouch, this is potentially a lot! But I guess that's true for top-level lookup as well.)